localflavor/bg/validators.py

```diff
@@ -1,36 +1,41 @@
 from django.core.exceptions import ValidationError
+from django.utils.deconstruct import deconstructible
 from django.utils.translation import ugettext_lazy as _
 
 from .utils import get_egn_birth_date
 
 
-def egn_validator(egn):
+@deconstructible
+class EGNValidator(object):
     """
     Check Bulgarian unique citizenship number (EGN) for validity.
 
     More details https://en.wikipedia.org/wiki/Unique_citizenship_number
     Full information in Bulgarian about algorithm is available here
     http://www.grao.bg/esgraon.html#section2
     """
-    def check_checksum(egn):
+
+    def check_checksum(self, egn):
```

Comment:
Can you mark new methods as private (prefixed with _)? This applies to all of the methods that you've created on the forms. I want to be clear that these are internal and not public. Please to this for data fields on the form as well.